docs/reference/data-streams/lifecycle/tutorial-manage-new-data-stream.asciidoc

```diff
@@ -93,19 +93,24 @@ The result will look like this:
 {
   "data_streams": [
     {
-      "name": "my-data-stream",<1>
+      "name": "my-data-stream",                                   <1>
       "lifecycle": {
-        "enabled": true,       <2>
-        "data_retention": "7d" <3>
+        "enabled": true,                                          <2>
+        "data_retention": "7d",                                   <3>
+        "effective_retention": "7d",                              <4>
+        "retention_determined_by": "data_stream_configuration"    <5>
       }
     }
   ]
 }
 --------------------------------------------------
 <1> The name of your data stream.
 <2> Shows if the data stream lifecycle is enabled for this data stream.
-<3> The retention period of the data indexed in this data stream, this means that the data in this data stream will
+<3> The desired retention period of the data indexed in this data stream, this means that if there are no other limitation
```

Comment:
`are` is plural and `limitation` is singular -- I think you want `if there are no other limitations`